scipy/linalg/tests/test_sketches.py

```diff
@@ -0,0 +1,72 @@
+"""Tests for _sketches.py."""
+
+from __future__ import division, print_function, absolute_import
+import numpy as np
+from scipy.linalg import clarkson_woodruff_transform
+
+__usage__ = """
```

Comment:
Better leave this out, it's mostly noise

Reply:
Done.